### ISSUE TITLE
QR Code Generator Not Working

### ISSUE THREAD

**Issue by dhanson358** [rbCAS/CASino#111]
Installed a clean copy of CASino.  Hooks up fine and let's me login, but when I try to enable two-factor authentication, I get the following error:

````
ActionView::Template::Error (code length overflow. (1052>864)):
    11:         <%= t('two_factor_authenticators.instructions') %>
    12:       </p>
    13:       <div id="qr-code">
    14:         <img src="<%= otp_qr_code_data_url(@two_factor_authenticator) %>" height="250" width="250"><br />
    15:       </div>
    16:       <p id="secret">
    17:         <%= t('two_factor_authenticators.secret') %>:
````

**Comment by pencil** [comment]
Thank you. Fixed in CASino 4.0.1.

**Comment by dhanson358** [comment]
Works great now, thanks very much!
